samza-core/src/main/scala/org/apache/samza/storage/SideInputsManager.java

```diff
@@ -507,4 +496,32 @@ private static boolean awaitSideInputTasks(Map<TaskName, CountDownLatch> sideInp
     }
     return true;
   }
+
+  private static class SideInputRunLoopConfig extends RunLoopConfig {
+
+    public SideInputRunLoopConfig(Config config) {
+      super(config);
+    }
+
+    @Override
+    public int getMaxConcurrency() {
```

Comment:
minor: Lets add comment here for someone in future to have the context:
Sideinput only processes single message per task and add the no windowing below